assets/large_language_models/rag/components/src/chat_flow_with_varients_mlindex.json

```diff
@@ -0,0 +1,413 @@
+              "type": [
+                "string"
+              ]
+            },
+            "chat_history": {
+              "name": "chat_history",
+              "type": [
+                "string"
+              ]
+            }
+          },
+          "code": "@@prompt_varient_1",
+          "is_builtin": false
+        },
+        {
+          "name": "Prompt_varients__variant_2",
+          "type": "prompt",
+          "inputs": {
+            "contexts": {
+              "name": "contexts",
+              "type": [
+                "string"
+              ]
+            },
+            "question": {
+              "name": "question",
+              "type": [
+                "string"
+              ]
+            },
+            "chat_history": {
+              "name": "chat_history",
+              "type": [
+                "string"
+              ]
+            }
+          },
+          "code": "@@prompt_varient_2",
+          "is_builtin": false
+        },
+        {
+          "name": "answer_the_question_with_context",
+          "type": "llm",
+          "inputs": {
+            "prompt_text": {
+              "name": "prompt_text",
+              "type": [
+                "string"
+              ]
+            }
+          },
+          "code": "{{prompt_text}}",
+          "is_builtin": false
+        }
+      ]
+    },
+    "nodeVariants": {
+      "embed_the_question": {
+        "defaultVariantId": "variant_0",
+        "variants": {
+          "variant_0": {
+            "node": {
+              "name": "embed_the_question",
+              "tool": "Embedding",
+              "inputs": {
+                "connection": "@@Embedding_Connection",
+                "deployment_name": "@@Embedding_Deployment_Name",
+                "input": "${flow.question}"
+              },
+              "reduce": false
+            }
+          }
+        }
+      },
+      "search_question_from_indexed_docs": {
+        "defaultVariantId": "variant_0",
+        "variants": {
+          "variant_0": {
+            "node": {
+              "name": "search_question_from_indexed_docs",
+              "tool": "Vector Search",
+              "inputs": {
+                "path": "@@MLIndex_Asset_Id",
+                "query": "${embed_the_question.output}",
+                "top_k": "2"
+              },
+              "reduce": false
+            }
+          }
+        }
+      },
+      "generate_prompt_context": {
+        "defaultVariantId": "variant_0",
+        "variants": {
+          "variant_0": {
+            "node": {
+              "name": "generate_prompt_context",
+              "tool": "generate_prompt_context",
+              "inputs": {
+                "search_result": "${search_question_from_indexed_docs.output}"
+              },
+              "reduce": false
+            }
+          }
+        }
+      },
+      "Prompt_varients": {
+        "defaultVariantId": "@@Primary_Metric_Name",
```

Comment:
Can you replace this with names like Variant_0, Variant_1 etc since we aare no longer doing autoprompting in these pipelines.

Reply:
fixed